com.microsoft.mrtk.graphicstools.unity/Runtime/Experimental/Magnifier/GrabScreenFeature.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+
+namespace Microsoft.MixedReality.GraphicsTools
+{
+    [ExecuteInEditMode]
+    public class GrabScreenFeature : ScriptableRendererFeature
```

Comment:
Question, is this render feature used anywhere? Or does the DrawFullScreenFeature take its place?